src/shared/utils/helpers/num-to-si.ts

```diff
@@ -6,5 +6,8 @@ const SHORTNUM_SI_FORMAT = new Intl.NumberFormat("en-US", {
 });
 
 export default function numToSI(value: number): string {
+  if (value < 10000) {
+     return value.toString()
```

Comment:
I wonder if it might be nice to use `toLocaleString()` to get the thousands separator in there?